source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,188 @@
+
+If a battery shows significantly lower voltages at the higher test current loads, it may not be done charging, or it may need to be retired.
+
+
+Understanding Battery Voltages
+------------------------------
+
+A "12V battery" is anything but 12V.
+
+Fully charged, a battery can be anywhere from 13 to 14 volts open circuit (Voc). Open circuit voltage is measured with *nothing* connected.
+
+Once a load (like a robot) is connected, and any amount of current is flowing, the battery voltage will drop. So if you check a battery with a Voltmeter, and it reads 13.5, and then connect it to your robot and power on, it will read lower, maybe 13.1 on the Driver Station display. Those numbers will vary with every battery and specific robot, see Characterization below. Once your robot starts running, it will pull more current, and the voltage will drop further.
+
+Depleted batteries will read between 11.5 and 12.5 volts on an idle robot. While batteries can be run lower, it will risk permanent battery damage.
+
+(SLA battery voltage and current performance also depends on temperature: cool batteries are happy batteries.)
+
+
+Battery Characterization
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A battery analyzer can be used to give a detailed inspection and comparison of battery performance.
+
+.. image:: images/robot-battery/cba-graph.jpg
+  :alt: Graph from a common battery analyzer plotting volts and AmpHrs for many different batteries.
+
+It will provide graphs of battery performance over time. This test takes significant time (roughly two hours) so it is less suited to testing during competition. It is recommended to run this test on each battery every year to monitor and track its performance. This will determine how it should be used: matches, practice, testing, or disposed of.
+
+At the standard 7.5 amps test load, competition batteries should have at least a 11.5 amp hour rating. Anything less than that should only be used for practice or other less demanding use cases.
+
+Most teams do not have Battery Analyzer setups.
+
+
+Battery Longevity
+^^^^^^^^^^^^^^^^^^^
+
+A battery is rated for about 1200 normal charge/recharge cycles. The high currents required for an FRC match reduce that lifespan to about 400 cycles. These cycles are intended to be relatively low discharge, from around 13.5 down to 11 or 12 volts. Deep cycling the battery (running it all the way down to less than 6 volts) will damage it.
+
+Batteries last the longest if they are kept fully charged when not it use, either by charging regularly or by use of a maintenance charger. Batteries drop roughly 0.1V every month of non-use.
+
+Batteries need to be kept away from both extreme heat and cold. This generally means storing the batteries in a climate controlled area, such as a classroomn closet, rather than a shipping container in a parking lot.
```

Comment:
climate controlled area is fine I don't understand why we need these extra examples of what a climate controlled area is.

Reply:
This guide is for high schoolers. I don't expect a high schooler to know that a classroom closet and a shipping container that they physically experience as temperate in the fall winter and spring are Significantly Different for battery storage over the summer.

Reply:
Maybe this is also a California Experience thing - our only remotely harsh climate is hitting >100F a few times over the summer, the rest of the year is ~50F-80F. So I wouldn't expect my high schoolers to notice or think about that a parking lot container gets extra hot in the summer while they're not on campus, whereas I've had pvc pipe components deform on me in the central valley when stored in a closed container on a jobsite.